svc-lighthouse-api/src/main/java/gov/va/vro/abddataaccess/service/FhirClient.java

```diff
@@ -57,7 +57,7 @@ public class FhirClient {
 
   @Autowired private LighthouseApiService lighthouseApiService;
 
-  private String lastUpdated = null;
+  private String lastUpdated = "";
```

Comment:
Initialize with empty String instead of null to simplify logic in `getLastUpdated()` and return empty String instead of null from that method.

Reply:
The purpose of setting it to NULL is to make it to be set ONLY once as a Singleton. getLastUpdated() is expected to return either an empty string or a valid FHIR accepted date string. It should never return a NULL.